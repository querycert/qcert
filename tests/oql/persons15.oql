/* Simple select-from-where */
select p
from p in Customers
order by p->age
