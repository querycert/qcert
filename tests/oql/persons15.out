[{"$class":["Customer"],"$data":{"name":"John Doe","age":{"$nat":32},"cid":{"$nat":123}}},{"$class":["Customer"],"$data":{"name":"Jane Doe","age":{"$nat":32},"cid":{"$nat":124}}},{"$class":["Customer"],"$data":{"name":"Jill Does","age":{"$nat":32},"cid":{"$nat":126}}},{"$class":["Customer"],"$data":{"name":"Jim Does","age":{"$nat":34},"cid":{"$nat":125}}},{"$class":["Customer"],"$data":{"name":"Joan Doe","age":{"$nat":34},"cid":{"$nat":127}}},{"$class":["Customer"],"$data":{"name":"James Do","age":{"$nat":35},"cid":{"$nat":128}}}]